.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+name: App CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 15.x
+
+    - name: Install dependencies 
+      run: yarn install
+
+    - name: Format code
+      run: yarn run format
```

Comment:
I wouldn't have the CI workflow format the code. It doesn't get checked back into the repo after it's been formatted. Also, we have a [pre-commit hook](https://github.com/internetarchive/iaux-typescript-wc-template/blob/main/package.json#L73-L77) that formats the code whenever you commit so all code going through CI should be formatted.